core/lib/spree/testing_support/sequences.rb

```diff
@@ -0,0 +1,9 @@
+require 'factory_girl'
```

Comment:
I think this file would be better under `testing_support/factories` (but I don't feel that strongly about it)